osmaxx-py/osmaxx/excerptexport/templates/excerptexport/partials/send_all_links.html

```diff
@@ -3,7 +3,7 @@
 <a href="mailto:?subject=Download%20map%20data%20of%20{{ extraction_order.excerpt.name }}&body=
 {% for file in extraction_order.output_files.all %}
 {% if not file.deleted_on_filesystem %}
-http://{{ host_domain }}{% url 'excerptexport:download' uuid=file.public_identifier %}%0D%0A
+{{ file.content_type }}: http://{{ host_domain }}{% url 'excerptexport:download' uuid=file.public_identifier %}%0D%0A
```

Comment:
see comment above and also: [getting-the-current-domain-for-full-urls](https://docs.djangoproject.com/en/1.8/ref/contrib/sites/#getting-the-current-domain-for-full-urls)

I think this should really be provided as batteries included with Django, but unfortunately is not. There are two solutions, I think are valid:

* use `{{ request.scheme }}` instead of [http](https://docs.djangoproject.com/en/dev/ref/request-response/#django.http.HttpRequest.scheme) or `{% if request.is_secure %}https://{% else %}http://{% endif %}{{ request.get_host }}...`
* write an own template tag, ie. full_url (see below for an untested example)

```python
from django.conf import settings
from django import template

register = template.Library()


class FULL_URLNode(template.defaulttags.URLNode):
    def render(self, context):
        full_url = super()
        # only one slash, the other one comes from the standard template tag
        protocol_type = 'https:/' if settings.SECURE_SSL_REDIRECT else 'http:/'
        full_url = protocol_type + full_url

        if self.asvar:
            context[self.asvar] = full_url
            return ''
        else:
            return full_url

@register.tag
def full_url(parser, token):
    full_url = template.defaulttags.url(parser, token)
    full_url.__class__ = FULL_URLNode

    return full_url
```